submissions/adeyemiJuwon/dom/product.js

```diff
@@ -0,0 +1,72 @@
+let quantity = document.getElementById("quantity");
+// let price = document.getElementById("price");
+let shippingCost = document.getElementById("shippingCost");
+let totalCost = document.getElementById("totalCost");
+let button = document.getElementById("btn");
+
+
+let itemPrice = 900;
```

Comment:
You can set item price here. You have to get the value from the html  just like `quantity`  @Samju200

Reply:
Yes
Thank You 
Done @ToluAkin